packages/valory/skills/registration_abci/payloads.py

```diff
@@ -34,33 +34,9 @@ def __str__(self) -> str:
         return self.value
 
 
+@dataclass(frozen=True)
 class RegistrationPayload(BaseTxPayload):
     """Represent a transaction payload of type 'registration'."""
 
+    initialisation: str
```

Comment:
Why did we change this away from `Optional[str]`?

Reply:
because we cannot serialize `None` using protobuf. Can revert this and move away from protobuf